sentry/src/main/java/io/sentry/ExternalOptions.java

```diff
@@ -150,6 +152,9 @@ public final class ExternalOptions {
     options.setCaptureOpenTelemetryEvents(
         propertiesProvider.getBooleanProperty("capture-open-telemetry-events"));
 
+    options.setEnableLogs(propertiesProvider.getBooleanProperty("logs.enabled"));
```

Comment:
could also be experimental

Reply:
We decided to not namespace options with `experimental`, see https://github.com/getsentry/sentry-java/pull/4390